<launch>
    <node name="sphero_stage" pkg="sphero_stage" type="start.py" output="screen"/>
    <node name="reynold" pkg="reynold" type="projectManager.py" output="screen"/>
    <node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" output="screen"/>
</launch>
